scripts/content-modules/adjust-pages.pl

```diff
@@ -15,6 +15,10 @@
 my $semConvRef = "$specRepoUrl/blob/main/semantic_conventions/README.md";
 my $spec_base_path = '/docs/reference/specification';
 my $path_base_for_github_subdir = "content/en$spec_base_path";
+my %versions = qw(
+  spec: 1.20.0
```

Comment:
This is manually updated for now, right?

Reply:
@tigrannajaryan - right, but I'll setup the auto-update in my next spec-related PR -- it's one of the tasks of #2387.

Reply:
Here's the PR:

- #2674